tests/e2e/package.json

```diff
@@ -30,7 +30,7 @@
     "axios": "0.18.1",
     "chai": "4.2.0",
     "chromedriver": "80.0.1",
-    "mocha": "6.1.4",
+    "mocha": "7.1.1",
```

Comment:
I haven't found a CQ for the mocha 7.1.1
![Screenshot from 2020-04-09 17-24-35](https://user-images.githubusercontent.com/22724619/78905665-06b63700-7a87-11ea-80d7-dcaa0bec99d8.png)

Reply:
it's a build and test dependency (devDependencies vs dependencies)

when we filed CQ it's including 'and later' versions so I would say there is no issue there

Reply:
@benoitf could you please point me to this, because as far as I remember we should do CQ for each version (ans a list of mocha CQs support this assumption).

Reply:
https://wiki.eclipse.org/Development_Resources/IP/Test_and_Build_Dependencies

Reply:
I think most of test and build dependencies have already been filed for dashboard